osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -495,5 +495,12 @@ protected override void Dispose(bool isDisposing)
         }
 
         #endregion
+
+        protected class MouseInputAbsorbingContainer : Container
```

Comment:
This is a little too generic to live here as `protected`. I'd consider extracting `ModSettingsContainer` to a separate component and apply the input handler overrides there.